Bug: v2.0.0 Beta - Clicking 'link a new account' is showing me the success screen of my last authentication

Hello,

In following the steps to migrate from v1 to v2, I had to update one of my bank accounts (got the error ` PlaidError: ITEM_LOGIN_REQUIRED` after running `mintable fetch`.
1. I ran `mintable account-setup`
2. clicked update on the outdated bank account
3. successfully completed the handshake with plaid. 
4. I then got back to the main screen (titled `Plaid – Account Configuration`) and clicked  `Link a new account`.
5. It then redirects me back to plaid and shows me the success screen on step 3.

seeing this in the terminal

```
2020-06-24T22:18:29.688Z [ERROR] Encountered error during authentication.

 {
  display_message: '',
  error_code: 'item-no-error',
  error_message: '',
  error_type: ''
}
```

I cannot link a new account.

6. i reran `mintable account-setup` and I see a duplicate row now for the bank account i updated in step 1-3. Pressing `link new account` now works correctly.

--- a/package.json
+++ b/package.json
@@ -7,8 +7,8 @@
     "preferGlobal": true,
     "scripts": {
         "build": "tsc",
-        "lint": "prettier --check ./src/**/*",
-        "lint-fix": "prettier --write ./src/**/*",
+        "lint": "prettier --check ./src/**/*.js ./src/**/*.html ./src/**/*.ts",
+        "lint-fix": "prettier --write ./src/**/*.js ./src/**/*.html ./src/**/*.ts",
         "test": "npm run-script build && npm run-script lint"
     },
     "files": [

--- a/src/integrations/plaid/add.html
+++ b/src/integrations/plaid/add.html
@@ -77,7 +77,9 @@
                     key: params.get('publicKey'),
                     product: ['transactions'],
                     onSuccess: function(public_token) {
-                        $.post('/get_access_token', { public_token: public_token })
+                        $.post('/get_access_token', { public_token: public_token }).then(resp =>
+                            setTimeout(() => location.reload(), 1000)
+                        )
                     },
                     onExit: function(error) {
                         // The user encountered a Plaid API error prior to exiting.
@@ -85,7 +87,9 @@
                             $.post('/get_access_token', { error: error })
                         }
                         // The user exited the Link flow.
-                        $.post('/get_access_token', { exit: true })
+                        $.post('/get_access_token', { exit: true }).then(resp =>
+                            setTimeout(() => location.reload(), 1000)
+                        )
                     }
                 }
 
@@ -99,17 +103,17 @@
                     })
                 })
 
-                $(document).on('click', 'button.update', function (e) {
-                    token = e.currentTarget.getAttribute("data-token")
+                $(document).on('click', 'button.update', function(e) {
+                    token = e.currentTarget.getAttribute('data-token')
                     console.log(token)
-                    $.post('/exchangeAccessToken', {token: token}).then(resp => {
+                    $.post('/exchangeAccessToken', { token: token }).then(resp => {
                         console.log(resp)
                         options.token = resp.token
                         console.log(options)
                         var handler = Plaid.create(options)
                         handler.open()
                     })
-                });
+                })
 
                 $('#link-button').on('click', function(e) {
                     var handler = Plaid.create(options)

--- a/src/integrations/plaid/plaidIntegration.ts
+++ b/src/integrations/plaid/plaidIntegration.ts
@@ -1,4 +1,4 @@
-import path from 'path' 
+import path from 'path'
 import { parseISO, format, subMonths } from 'date-fns'
 import plaid, { TransactionsResponse } from 'plaid'
 import { Config, updateConfig } from '../../common/config'
@@ -73,7 +73,11 @@ export class PlaidIntegration {
                 } else if (req.body.exit !== undefined) {
                     resolve(logInfo('Plaid authentication cancelled.'))
                 } else {
-                    reject(logError('Encountered error during authentication.', req.body.error))
+                    if ((req.body.error['error-code'] = 'item-no-error')) {
+                        resolve(logInfo('Account is OK, no further action is required.'))
+                    } else {
+                        reject(logError('Encountered error during authentication.', req.body.error))
+                    }
                 }
                 return res.json({})
             })
@@ -108,7 +112,9 @@ export class PlaidIntegration {
                 return server.close()
             })
 
-            app.get('/', (req, res) => res.sendFile(path.resolve(path.join(__dirname, '../../../src/integrations/plaid/add.html'))))
+            app.get('/', (req, res) =>
+                res.sendFile(path.resolve(path.join(__dirname, '../../../src/integrations/plaid/add.html')))
+            )
 
             server = require('http')
                 .createServer(app)

